sql/hostname.cc

```diff
@@ -513,42 +513,48 @@ int ip_to_hostname(struct sockaddr_storage *ip_storage,
 
   DBUG_EXECUTE_IF("getnameinfo_error_noname",
                   {
-                    strcpy(hostname_buffer, "<garbage>");
+                    safe_strcpy(hostname_buffer, sizeof(hostname_buffer),
+                    "<garbage>");
```

Comment:
Indentation is wrong here. The string should be aligned with the opening `(` on the previous row. Same as the other DBUG_EXECUTE_IFs.

Reply:
thanks @cvicentiu, just to avoid pushing commits back and forth I'll just ask here.
Is this the indentation you had in mind?
```
  DBUG_EXECUTE_IF("getnameinfo_error_noname",
                  {
                    safe_strcpy(hostname_buffer, sizeof(hostname_buffer),
                                "<garbage>");
                    err_code= EAI_NONAME;
                  }
                  );
```